setup.cfg

```diff
@@ -23,7 +23,7 @@ branch = main
 install_requires =
 	ewms-pilot==0.19.0 # the extras here are the same as 'oms-mqclient', so just define one place
 	healpy
-	icecube-skyreader
+	icecube-skyreader>=1.4.2
```

Comment:
This doesn't need to be pinned. Any fresh build of the project will always pull the latest.

Reply:
That's true, however I guess it wouldn't hurt to be explicit here? Also I'm not sure what would happen in the case where an older version of the dependency is already installed and this isn't specified.